ui/plugins/controller.js

```diff
@@ -301,6 +301,18 @@ treeherder.controller('PluginCtrl', [
             return thPinboard.count.numPinnedJobs;
         };
 
+        $scope.getCountPinnedTitle = function() {
+            var title = "";
+
+            if (thPinboard.count.numPinnedJobs === 1) {
+                title = "You have " + thPinboard.count.numPinnedJobs + " job pinned";
+            } else if (thPinboard.count.numPinnedJobs > 1) {
+                title = "You have " + thPinboard.count.numPinnedJobs + " jobs pinned";
+            }
+
+            return title;
+        };
+
```

Comment:
@camd this is that new function for the pin count tooltip/title. It seems to work fine in local testing. I chose to not set a var for it eg. `var pinCount = thPinboard.count.numPinnedJobs` to save a line. I did try that, and it seemed to make the pin count changes in the UI slightly slower. Anyway, I just stuck with the pinboard service everywhere.